src/controllers/dashboard/encodingsettings.js

```diff
@@ -19,6 +19,7 @@ function loadPage(page, config, systemInfo) {
     page.querySelector('#chkHardwareEncoding').checked = config.EnableHardwareEncoding;
     page.querySelector('#chkAllowHevcEncoding').checked = config.AllowHevcEncoding;
     page.querySelector('#chkAllowAv1Encoding').checked = config.AllowAv1Encoding;
+    page.querySelector('#chkAllowMjpegEncoding').checked = config.AllowMjpegEncoding;
```

Comment:
I thought this and L131 was removed before?

Reply:
Rebase issue, that one is out of tree for a long time and only merged 9 hours ago.